changelogs/config.yaml

```
@@ -0,0 +1,29 @@
+changelog_filename_template: ../CHANGELOG.rst
+changelog_filename_version_depth: 0
+changes_file: ../changelog.yaml
```

That file should stay there, otherwise the ACD changelog generator won't find it.
```suggestion
changes_file: changelog.yaml
```